app/models/blueprint.rb

```diff
@@ -0,0 +1,51 @@
+class Blueprint < ApplicationRecord
+  has_many :service_templates
+  private  :service_templates, :service_templates=
```

Comment:
@bzwei Is there a requirement to hide the `service_templates` relationship?  What is the concern here?